design/BulkAPI.md

```diff
@@ -119,7 +127,10 @@ GET /bulk?job_id=123e4567-e89b-12d3-a456-426614174000
   "processed_experiments": 23,
   "job_id": "54905959-77d4-42ba-8e06-90bb97b823b9",
   "job_start_time": "2024-10-10T06:07:09.066Z",
-  "job_end_time": "2024-10-10T06:07:17.471Z"
+  "job_end_time": "2024-10-10T06:07:17.471Z",
+  "webhook": {
+    "status": "COMPLETED"
+  }
```

Comment:
Are we having a single webhook or multiple currently? In the future are we supporting more? Please name it appropriately

Reply:
Yes, currently, there is a single webhook per JobID. I can make changes if support for multiple webhooks is needed.